source/agora/consensus/validation/Block.d

```diff
@@ -616,11 +610,7 @@ version (unittest)
         return isInvalidReason(block, engine, prev_height, prev_hash, findUTXO,
             checkFee, findEnrollment, enrolled_validators,
             prev_time_offset, (curr_time_offset == ulong.max) ? block.header.time_offset : curr_time_offset,
-            block_time_tolerance,
-            (in Transaction[] tx_set, in uint[] missing_validators)
-            {
-                return (Transaction[]).init;
-            });
```

Comment:
Why did this check get deleted?

Reply:
Because we don't pass the delegate to that function anymore, so we don't need this plumbing.